### ISSUE TITLE
Semantic3D fails to generate the prediction

### ISSUE THREAD

**Issue by yaoyao1900** [aboulch/ConvPoint#17]
Hi, @aboulch , Thanks for your great work!

My dataset is in the same format as semantic 3d. I had followed the steps for semantic3d. But the predicted Benchmark is failed. 

semantic3d_seg.py

`python semantic3d_benchmark_gen.py --rootdir, path to processed test npy file, --savedir, path to the state_dict.pth file, --test`

This Test process takes a long time and I have got a new test npy file. The benchmark in this npy file is all 0.

semantic3d_benchmark_gen.py

`python semantic3d_benchmark_gen.py --testdir, path to the original test data, --savedir, path to save benchmark .labels file, --refdata, path to processed test txt file, --reflabel, path to processed test npy file or path new generated test npy file`

This process is very fast. It's not same as you described. I got a predicted .labels file and all the benchmark in this .labels file is 1.

Thank you. I am waiting for your answer.
